projects/GKCore/GKCore/LangMan.cs

```diff
@@ -1408,7 +1408,7 @@ public static class LangMan
             /* 516 */ "Search unlinked namesakes",
             /* 517 */ "Check connection of families",
             /* 518 */ "Check database",
-            /* 519 */ "Search the patriarchs",
+            /* 519 */ "Find the patriarchs",

```

Comment:
A menu item. Probably original is more appropriate. I need to research about "find" vs "search" semantic.